examples/models/role_playing_with_mistral.py

```diff
@@ -0,0 +1,139 @@
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+# Licensed under the Apache License, Version 2.0 (the “License”);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an “AS IS” BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+
+from typing import List
+
+from colorama import Fore
+
+from camel.agents.chat_agent import FunctionCallingRecord
+from camel.configs import MistralConfig
+from camel.models import ModelFactory
+from camel.societies import RolePlaying
+from camel.toolkits import (
+    MATH_FUNCS,
+    SEARCH_FUNCS,
+)
+from camel.types import ModelPlatformType, ModelType
+from camel.utils import print_text_animated
+
+
+def main(
+    model_platform=ModelPlatformType.MISTRAL,
+    model_type=ModelType.MISTRAL_LARGE,
+    chat_turn_limit=10,
```

Comment:
IMO, Is it better to move it into a function?

Reply:
This is example code we don't need to add more abstraction for example code